challengeFive.js

```diff
@@ -0,0 +1,46 @@
+// First attempt 
+
+let cardNumber = "";
+
+function hideCardNumber(number){
+    const cardNumberString = number.toString();
+    const integers = cardNumberString.split("");
```

Comment:
Why is this called integers? It should be called cardNumberArray or something more descriptive.

Also, this could be done in one line such as const integers = number.toString().split("");

Save a line instead of having lines 6 and 7. You can do both in one line.